### PR TITLE
fix(FEC-12648): Text style configuration is not working

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

solves: FEC-12648

**Findings:**

**1.** by Default, the player uses its own text displayer and not the browser's native one
**2.** there are two level of text style (in PKTextConfigObject type):

  **textTrackDisplaySetting: PKTextTrackDisplaySettingObject,
  textStyle: PKTextStyleObject,forceCenter: boolean,**

**3.** by default both of them work
**4.** in case you use the browser native text tracks (useNativeTextTrack set to true) - only textStyle works
**5.** in case you use Shaka text displayer (useShakaTextTrackDisplay set to true) - none of them work

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [x] Docs have been updated

### CODE REVIEW

File: src/track/text-style.js [hunk 1]
```diff
@@ -257,9 +257,20 @@ class TextStyle {
     attributes.push('color: ' + TextStyle.toRGBA(this.fontColor, this.fontOpacity));
     attributes.push('background-color: ' + TextStyle.toRGBA(this.backgroundColor, this.backgroundOpacity));
     attributes.push('text-shadow: ' + this.getTextShadow());
+    attributes.push('font-size: ' + this.fontSize);
     return attributes.join('!important; ');
   }
 
+  toCSSObject(): string {
```
Comment (SivanA-Kaltura) [inline src/track/text-style.js:264]: where is this used ?

Comment (SivanA-Kaltura) [reply src/track/text-style.js]: so the bug was with the font size only ?

Comment (JonathanTGold) [reply src/track/text-style.js]: Right

File: flow-typed/types/text-style.js [hunk 2]
```diff
@@ -1,11 +1,23 @@
 // @flow
+
+/**
+ * @typedef {Object} PKTextStyleObject
+ * @property {"50%" | "75%" | "100%" | "200%" | "3000%" | "4000%"} fontSize='100%' - Percentage unit relative to the parent element's font size.
+ * @property {-2 | -1 | 0 | 2 | 3 | 4} fontScale=0 - - Integer number representing the scaling factor relative to the parent element's font size..
```
Comment (giladna) [inline flow-typed/types/text-style.js:6]: @JonathanTGold  please add note what are the defaults

**Review by giladna — CHANGES_REQUESTED** [review]

File: flow-typed/types/text-style.js (hunk 2 above)
Comment (SivanA-Kaltura) [reply flow-typed/types/text-style.js:6]: @giladna comments here, not in the flow type https://github.com/kaltura/playkit-js/blob/master/src/track/text-style.js

File: flow-typed/types/text-style.js [hunk 3]
```diff
@@ -1,11 +1,23 @@
 // @flow
+
+/**
+ * @typedef {Object} PKTextStyleObject
+ * @property {"50%" | "75%" | "100%" | "200%" | "3000%" | "4000%"} fontSize='100%' - Percentage unit relative to the parent element's font size.
```
Comment (SivanA-Kaltura) [inline flow-typed/types/text-style.js:5]: is it 3000,4000 or 300,400 ?

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: flow-typed/types/text-style.js (hunk 3 above)
Comment (JonathanTGold) [reply flow-typed/types/text-style.js]: fixed

**Review by giladna — APPROVED** [review]

**Pull request merged**